.changeset/late-chicken-carry.md

```diff
@@ -0,0 +1,7 @@
+---
+'@cypress-design/icon-registry': minor
+'@cypress-design/react-icon': minor
+'@cypress-design/vue-icon': minor
```

Comment:
Hmm, is there any documentation on the semantic versioning? I feel like new icons should really be a minor version. What's the reasoning behind using a patch version for new functionality?

https://semver.org/#summary

Reply:
I'm honestly not sure. I used minor as well before, and I got requested same change from reviewers. I think @emilyrohrbough or @warrensplayer should be able to explain this better.

Reply:
the [readme](https://github.com/cypress-io/cypress-design/blob/main/ReadMe.md?plain=1#L107) also says to use a minor version

Reply:
I wonder if the `patch` recommendation was when the projects were using the `0.x.x` versioning which would've definitely been problematic if a `minor` version was used since that would've been considered a breaking change and `yarn` wouldn't have automatically updated it.

Reply:
Ok, I see. I'll approve the PR.